From a0a44c0f5f59d1cfa6a5c06778927265fddab89d Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Fri, 28 Jan 2022 17:14:51 +0800
Subject: [PATCH] Set selinux policy type for wrlinux

Set var_selinux_policy_name to wr-mls for WRLinux LTS24

Upstream-Status: Inappropriate [WRLinux specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 .../guide/system/selinux/selinux_policytype/oval/shared.xml     | 2 +-
 linux_os/guide/system/selinux/var_selinux_policy_name.var       | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/linux_os/guide/system/selinux/selinux_policytype/oval/shared.xml b/linux_os/guide/system/selinux/selinux_policytype/oval/shared.xml
index 6a51b5ea94..348bdafc66 100644
--- a/linux_os/guide/system/selinux/selinux_policytype/oval/shared.xml
+++ b/linux_os/guide/system/selinux/selinux_policytype/oval/shared.xml
@@ -23,7 +23,7 @@
 
   <ind:textfilecontent54_object id="obj_selinux_policy" version="1">
     <ind:filepath>/etc/selinux/config</ind:filepath>
-    <ind:pattern operation="pattern match">^SELINUXTYPE=([\w]*)[\s]*$</ind:pattern>
+    <ind:pattern operation="pattern match">^SELINUXTYPE=([^\s]*)[\s]*$</ind:pattern>
     <ind:instance datatype="int">1</ind:instance>
   </ind:textfilecontent54_object>
 </def-group>
diff --git a/linux_os/guide/system/selinux/var_selinux_policy_name.var b/linux_os/guide/system/selinux/var_selinux_policy_name.var
index 891c5818fd..07be509858 100644
--- a/linux_os/guide/system/selinux/var_selinux_policy_name.var
+++ b/linux_os/guide/system/selinux/var_selinux_policy_name.var
@@ -7,6 +7,7 @@ description: |-
     <br />targeted - Only targeted network daemons are protected.
     <br />strict - Full SELinux protection.
     <br />mls - Multiple levels of security
+    <br />wr-mls - Multiple levels of security on Wind River Linux
 
 type: string
 
@@ -18,3 +19,4 @@ options:
     default: targeted
     mls: mls
     targeted: targeted
+    wr-mls: wr-mls
-- 
2.25.1

